renderdoccmd/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ elseif(UNIX)
         list(APPEND libraries PRIVATE -lX11-xcb)
     endif()
 
+    if(ENABLE_WAYLAND)
+        list(APPEND libraries PRIVATE -lwayland-client)
+        list(APPEND libraries PRIVATE -lwayland-cursor)
```

Comment:
Why is the cursor library needed here? The renderdoccmd preview doesn't have any mouse interaction.